make_manifest.py

```diff
@@ -141,30 +152,33 @@ def collect_icons_for_alexa_top(count, extra_domains=None):
             # Retry with http
             url = 'http://{hostname}'.format(hostname=hostname)
             icons = fetch_icons(url)
+        best_icon_url = get_best_icon(minwidth, icons)
         results.append({
             'hostname': hostname,
             'url': url,
             'icons': icons,
             'rank': rank,
-            'best_icon': get_best_icon(icons)
+            'best_icon': best_icon_url
         })
     logging.info('Done fetching icons')
     return results
 
 
 @click.command()
-@click.option('--count', default=10, help='Number of sites from Alexa Top Sites')
+@click.option('--count', default=10, help='Number of sites from a list of Top Sites that should be used to generate the manifest. Default is 10.')
+@click.option('--topsitesfile', type=click.Path(exists=True), help='A csv file containing comma separated rank and domain information (in the same order) of the Top Sites. If no file is provided then Alexa Top Sites are used.')
```

Comment:
Shall we make this argument optional? Since it will fall back to Alexa if not specified.

Reply:
The option `--topsitesfile` is optional in the sense if the user doesn't specify it on command line, it is not used. Is this what you meant?

Reply:
Yep, I didn't see a default value here, also noticed the path check, so just wanted to make sure it's ok to omit this argument.